plugins/image-prioritizer/class-image-prioritizer-img-tag-visitor.php

```diff
@@ -22,16 +22,39 @@ final class Image_Prioritizer_Img_Tag_Visitor extends Image_Prioritizer_Tag_Visi
 	/**
 	 * Visits a tag.
 	 *
+	 * @since 0.1.0
+	 * @since n.e.x.t Separate the processing of <img> and <picture> elements.
+	 *
 	 * @param OD_Tag_Visitor_Context $context Tag visitor context.
 	 *
 	 * @return bool Whether the tag should be tracked in URL Metrics.
 	 */
 	public function __invoke( OD_Tag_Visitor_Context $context ): bool {
 		$processor = $context->processor;
-		if ( 'IMG' !== $processor->get_tag() ) {
+		$tag       = $processor->get_tag();
+
+		if ( 'PICTURE' === $tag ) {
+			return $this->process_picture( $processor, $context );
+		}
+
+		if ( 'IMG' !== $tag ) {
 			return false;
 		}
 
+		return $this->process_img( $processor, $context );
+	}
```

Comment:
Might as well refactor this a bit so the cases are consistent:

```suggestion
		} elseif ( 'IMG' === $tag ) {
			return $this->process_img( $processor, $context );
		}

		return false;
	}
```